utils/service/txmgr/mocks/TxManager.go

```diff
@@ -0,0 +1,77 @@
+// Code generated by mockery v2.26.0. DO NOT EDIT.
+
+package mocks
+
+import (
+	context "context"
+
+	common "github.com/ethereum/go-ethereum/common"
+
+	mock "github.com/stretchr/testify/mock"
+
+	txmgr "github.com/kroma-network/kroma/utils/service/txmgr"
+
+	types "github.com/ethereum/go-ethereum/core/types"
+)
+
+// TxManager is an autogenerated mock type for the TxManager type
+type TxManager struct {
+	mock.Mock
+}
+
+// From provides a mock function with given fields:
+func (_m *TxManager) From() common.Address {
```

Comment:
Do you know why optimism uses `_m` instead of `m`?

Reply:
also generated code by mockery